packages/appsync-emulator-serverless/README.md

```diff
@@ -25,6 +25,24 @@ This package will download and run the dynamodb emulator as part of it's appsync
 # NOTE unless you assign a specific port a random one will be chosen.
 yarn appsync-emulator --port 62222
 ```
+#### dynamodb with fixed port
+
+optional start dynamodb at a fixed port - e.g. 8000
+```sh
+# NOTE unless you assign a specific port for dynamodb a random one will be chosen.
+yarn appsync-emulator --port 62222 --dynamodb-port 8000
+```
+to access the dynamodb instance you need to use the following configuration:
```

Comment:
Do you mind changing this to ?

`to access the dynamodb instance using javascript`..

and including 

`const { DynamoDB } = require('aws-sdk');`

?